R/translate-sql-window.R

```diff
@@ -122,10 +122,9 @@ rows <- function(from = -Inf, to = 0) {
   }
 }
 
-
 #' @rdname win_over
 #' @export
-win_rank <- function(f) {
+win_rank <- function(f, use_default_order_null = FALSE) {
```

Comment:
This argument name feels a bit long to me, and needs to be documented. Maybe `empty_order = TRUE` or  `order_by = c("required", "optional")`, or something along those lines?

Reply:
Changed to `empty_order` (defaulting to `FALSE` to not apply to all backends) and added some docs!